test/verify/check-metrics

```diff
@@ -1152,7 +1153,7 @@ class TestGrafanaClient(MachineCase):
             # expect a "Load average" panel with a sensible number
             max_load = bg.text("div:contains('Load average') .graph-legend-series:contains('1 minute') .max")
             self.assertGreater(float(max_load), 0)
-        except Error:
+        except Exception:
```

Comment:
What the linters don't like is just `except:` without any type, but `except Exception` is fine, even without raise. And why wouldn't it? :-)

Reply:
They probably accept it because it gets re-thrown. without an unconditional `raise` this should never be done, as it's the exact same blanket exception as a bare `except:` (as *every* exception is derived from class `Exception`).

Anyway, this was just thinking aloud.